ds3-sdk/src/main/java/com/spectralogic/ds3client/commands/HeadBucketResponse.java

```diff
@@ -58,7 +58,7 @@ private void setStatus(final int statusCode) {
             case 403: this.status = Status.NOTAUTHORIZED; break;
             case 404: this.status = Status.DOESNTEXIST; break;
             default: {
-                LOG.error("Unexpected status code: " + Integer.toString(statusCode));
```

Comment:
didn't we already do this?  I better go make sure that commit got merged to 3_2_autogen branch